templates/course_info.html

```diff
@@ -43,42 +43,72 @@ <h2 class="sub-header">Assistant(s) for this year ({{ course.year }} - {{ course
-                        </table>
-                    </div>
-                    <!-- Evaluations Pane -->
-                    <div class="tab-pane fade show" id="evaluations" role="tabpanel" aria-labelledby="evaluations-tab">
-                        <!-- Content for Evaluations -->
-                        <br>
-                        <p>Content for evaluations will go here.</p>
+                        <!-- Tab panes -->
+                        <div class="tab-content">
+                            <!-- Assistants Pane -->
+                            <div class="tab-pane fade show active" id="assistants-{{ course.year }}" role="tabpanel"
+                                 aria-labelledby="assistants-tab-{{ course.year }}">
+                                <table class="table table-hover mt-3">
+                                    <thead>
+                                    <tr>
+                                        <th>Last name</th>
+                                        <th>First name</th>
+                                    </tr>
+                                    </thead>
+                                    <tbody>
+                                    </tbody>
+                                </table>
+                            </div>
+                            <!-- Evaluations Pane -->
+                            <div class="tab-pane fade show" id="evaluations-{{ course.year }}" role="tabpanel"
+                                 aria-labelledby="evaluations-tab-{{ course.year }}">
+                                <!-- Content for Evaluations -->
+                                <table class="table table-hover" id="evaluationTable">
+                                    <thead>
+                                    <tr>
+                                        <th>Course name</th>
+                                        <th>Year</th>
+                                        <th>User</th>
+                                    </tr>
+                                    </thead>
+                                    <tbody>
+                                    {% if evaluations is none %}
+                                        <tr>
+                                            <td colspan="3">No evaluations for this year.</td>
+                                        </tr>
+                                    {% endif %}
+                                    {% for evaluation in evaluations %}
+                                        <tr>
+                                            <td><a href="{{ url_for('course.course_evaluation', evaluation_id=evaluation.id) }}">
+                                                {{ evaluation.course.code }} - {{ evaluation.course.title }}</a>
+                                            </td>
+                                            <td>{{ evaluation.course.year }}</td>
+                                            <td>{{ evaluation.user.name }} {{ evaluation.user.first_name }}</td>
```

Comment:
In the `evaluations.html` template it is mentioned that evaluations are done anonymously. This should be coherent. Besides, if the name can be shown, it would be useful to have on the page showing the read-only evaluation too.